src/App.css

```diff
@@ -17,3 +17,57 @@
   padding: 0;
   font-family: 'Tilt Neon', sans-serif;
 }
+
+#section-one {
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  text-align: center;
+  background-color: #9e2281;
+  opacity: 0.5;
+}
+
+#section-two {
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  text-align: center;
+  background-color: #77229e;
+  opacity: 0.5;
+}
+
+#section-three {
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  text-align: center;
+  background-color: #3d229e;
+  opacity: 0.5;
+}
+
+.animate-me {
```

Comment:
CSS selectors, in this case .animate-me, should be descriptive. I don't know what .animate-me is.